nixos/modules/services/hardware/logiops.nix

```diff
@@ -1,104 +1,28 @@
 { config, lib, pkgs, ... }:
 
 let
-  inherit (lib) mdDoc mkEnableOption mkIf mkOption types;
+  inherit (lib) mkEnableOption mkIf mkOption types;
   cfg = config.services.logiops;
+  format = pkgs.formats.logiops { };
```

Comment:
Just for clarification, is this supposed to be `pkgs.formats.logiops` or `pkgs.formats.libconfig`?

Reply:
It's supposed to be `pkgs.formats.libconfig`. Libconfig is the configuration languages of choice that is used to configure `logiops` (https://github.com/PixlOne/logiops/wiki/Configuration)